### DISCUSSION TITLE
Editor - Allow transformation of perfectly stacked objects.

### DISCUSSION THREAD

**Discussion by mojobojo** [ppy/osu#31272] [Ideas]
I had a bit of confusion in the editor today as I was transforming 3 note bursts and came across a burst I couldn't transform. After a bit of trial and error I discovered it was my stack leniency. I think it would be good to allow us to transform a perfect stack and when we do, treat it as if it were layed out that way rather than the stack leniency making it look that way.

This is what I expected to see:
![image](https://github.com/user-attachments/assets/4c5c15b8-2cc6-4f06-b063-401bfb6d5057)

This is what I was confused by:
![image](https://github.com/user-attachments/assets/9ff65848-a57c-4ac9-92e9-7d328f291cbe)

**Comment by peppy** [comment]
Logically this doesn't make sense because stacking doesn't affect the actual hit locations. I can understand what you're trying to achieve here and maybe it makes sense to support this, but I'm not sure how easy that will be.

**Reply by mojobojo** [reply]
If this isn't possible, at the very least I would like to point out that it was just a point of confusion coming from me as an inexperienced mapper. Possibly just something to make it more clear that what I am looking the same position hit location wise but the stack leniency makes it appear different. This may just be me being oblivious because maybe I should have assumed no transform controls means something else.
